### PR TITLE
Update action.yml to use Packj source code instead of PyPi

### PR DESCRIPTION
**Pull request by KhoiUna**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -10,46 +10,48 @@ branding:
   icon: "package" # icon name from Feather open source icons pack
   color: "orange"
 
-permissions:
```
Comment (ashishbijlani) [inline action.yaml:13]: Removing permissions is **NOT** a good idea. Please add `read` permissions.

Comment (KhoiUna) [reply action.yaml:13]: `permissions` is not supposed to be in `action.yaml`. VS code warns me that

File: action.yaml [hunk 2]
```diff
@@ -10,46 +10,48 @@ branding:
...
+      shell: bash
+      run: source venv/bin/activate
+
+    - name: Install Packj's dependencies
+      shell: bash
+      run: pip3 install -r ./packj/requirements.txt
+
     - name: Auditing deps with Packj
       shell: bash
-      run: |
```
Comment (ashishbijlani) [inline action.yaml:33]: We need correct path to dependencies. That's exactly what this piece of code does. Please add it back.

**Review by ashishbijlani — CHANGES_REQUESTED** [review]
Please make the suggested changes. Thanks!

**Comment by KhoiUna** [comment]
I have made the changes

File: action.yaml [hunk 3]
```diff
@@ -50,7 +50,24 @@ runs:
...
+              dep_file=$(echo $item | cut -f2 -d:)
+              path=${{ github.workspace }}/$dep_file
+              if [ -f $path ]; then
+                input_files+=$pm_name":"$path,
+              fi
+            fi
+          done
+        if [ ! -z input_files ]; then
+          input=$(echo $input_files | sed 's/,/ /g')
+          python3 ./packj/main.py audit -f $input
```
Comment (ashishbijlani) [inline action.yaml:69]: Don't call `packj/main.py` from here. Just call `main.py` in the top-level dir.

**Review by ashishbijlani — DISMISSED** [review]

**Review by ashishbijlani — APPROVED** [review]
Looks good.

**Pull request merged**